public/docs/_examples/toh-6/ts/app/hero-search.component.css

```diff
@@ -1,11 +1,4 @@
 /* #docregion */
```

Comment:
I don't know why git decided this was a rename operation when I rebased, but I don't think that hurts anything.